src/components/DataBar/DataBar.stories.tsx

```diff
@@ -131,7 +131,7 @@ export const InteractiveExample: StoryObj<Args> = {
   parameters: {
     /* 1 */
     chromatic: {
-      skip: true,
```

Comment:
I happened to notice this while looking for how to disable the chromatic snapshots because I can never remember what the property is called.